wallet/src/ui/app/KeypairVault.ts

```diff
@@ -3,13 +3,11 @@
 
 import { Ed25519Keypair } from '@mysten/sui.js';
 
-import { getKeypairFromMnemonics } from '_shared/cryptography/mnemonics';
-
 export default class KeypairVault {
     private _keypair: Ed25519Keypair | null = null;
 
     public set mnemonic(mnemonic: string) {
-        this._keypair = new Ed25519Keypair(getKeypairFromMnemonics(mnemonic));
+        this._keypair = Ed25519Keypair.deriveKeypair("m/44'/784'/0'/0'/0'", mnemonic);
```

Comment:
this switches over the keypair vault to actually use the derive (with a default path)